.github/workflows/pr-labeler.yml

```diff
@@ -13,6 +16,16 @@ jobs:
       pull-requests: write
       issues: write
     steps:
-    - run: gh pr edit ${{ github.event.pull_request.number }} --add-label "auto-approve" -R ${{ github.repository }}
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: gh pr edit ${{ github.event.pull_request.number }} --add-label "auto-approve" -R ${{ github.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  copy-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: kaizen3031593/pr-triage-manager@main
```

Comment:
I am in the process of trying to get approval to `aws-github-ops`, since our [`aws-issue-triage-manager`](https://github.com/aws-github-ops/aws-issue-triage-manager) lives there. But for now, since we want this workflow asap, this is something we're going to have to live with.